helpers/pause.js

```diff
@@ -1,7 +1,7 @@
 const readlineSync = require('readline-sync');
 
 let pause = () => {
-    readlineSync.question('Press any key to continue ...');
+    readlineSync.keyInPause();
```

Comment:
This does not take enter key 😕, also `Ctrl + C` (cancel) doesn't work.

Moved this to enter key only in https://github.com/siddharthkp/learn-es6/commit/ef6e9e033fc30a56febc4913f2fad1466d069319